libs/island-ui/core/src/lib/Swiper/Swiper.tsx

```diff
@@ -27,13 +36,17 @@ export const Swiper: FC<React.PropsWithChildren<unknown>> = ({ children }) => {
   return (
     <div className={styles.root}>
       <div className={cn(styles.container)} ref={ref}>
-        <div className={styles.slides}>
+        <div className={cn(styles.slides)}>
           {arr.map((child, i) => (
             <div
               key={i}
-              className={styles.slide}
+              className={cn(
+                styles.slide,
+                className,
+                i === 0 && styles.noMargin,
+              )}
```

Comment:
```suggestion
              className={cn(
                styles.slide,{
                [className]: !!className,
                [styles.noMargin]: i === 0,
              })}
```
otherwise you'll get styles like class="slide undefined false"